.github/workflows/adhoc-release.yml

```
@@ -0,0 +1,91 @@
+      - uses: oven-sh/setup-bun@4bc047ad259df6fc24a6c9b0f9a0cb08cf17fbe5 # v2
+
+      - name: Install jq
+        run: sudo apt-get install jq
+
+      - name: Bump version
+        env:
+          GH_TOKEN: ${{ secrets.GH_ACCESS_TOKEN }}
+          RELEASE: 1
+          INPUT_BRANCH: ${{ inputs.branch }}
+          INPUT_VERSION: ${{ inputs.version }}
+        run: "./scripts/bump_version.sh $INPUT_BRANCH $INPUT_VERSION"
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2086:info:1:27: Double quote to prevent globbing and word splitting

(shellcheck)